components/social/ShareButtons.js

```diff
@@ -3,6 +3,8 @@ import './social-buttons.scss';
 import React from 'react';
 
 import Icon from 'components/common/ui/Icon';
+import ButtonGroup from 'components/common/ButtonGroup/ButtonGroup';
```

Comment:
lets store it just like 
```suggestion
import ButtonGroup from 'components/common/ButtonGroup';
```